pkg/lvgl/contrib/lvgl.c

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (C) 2019 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     pkg_lvgl
+ * @{
+ *
+ * @file
+ * @brief       LittlevGL glue code
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ * @}
+ */
+
+#include "thread.h"
+
+#include "xtimer.h"
+#include "log.h"
+
+#include "lvgl/lvgl.h"
+#include "lv_conf.h"
+#include "lvgl_riot.h"
+#include "disp_dev.h"
+
+#ifndef LVGL_TASK_THREAD_PRIO
+#define LVGL_TASK_THREAD_PRIO   (THREAD_PRIORITY_MAIN - 1)
+#endif
+
+#ifndef LVGL_COLOR_BUF_SIZE
+#define LVGL_COLOR_BUF_SIZE     (LV_HOR_RES_MAX * 5)
+#endif
+
+#ifndef LVGL_ACTIVITY_PERIOD
+#define LVGL_ACTIVITY_PERIOD    (5 * MS_PER_SEC)
+#endif
+
+#ifndef LVGL_TASK_HANDLER_DELAY
+#define LVGL_TASK_HANDLER_DELAY (5 * US_PER_MS)
+#endif
+
+static char _task_thread_stack[THREAD_STACKSIZE_MAIN];
```

Comment:
This should be "THREAD_STACKSIZE_DEFAULT" (or maybe _MEDIUM or _LARGE?).
A user might have a large main and thus overrides "THREAD_STACKSIZE_MAIN", which shouldn't affect lvgl's thread.

Reply:
Changed to THREAD_STACKSIZE_LARGE, otherwise it doesnt fit.